relay/paseo/constants/src/lib.rs

```diff
@@ -106,19 +106,42 @@ pub mod xcm {
         // Preallocated for the Root body.
         #[allow(dead_code)]
         const ROOT_INDEX: u32 = 0;
-        // The bodies corresponding to the Polkadot OpenGov Origins.
+        // The bodies corresponding to the Paseo OpenGov Origins.
         pub const FELLOWSHIP_ADMIN_INDEX: u32 = 1;
+        // The body corresponding to the Treasurer OpenGov track.
+        pub const TREASURER_INDEX: u32 = 2;
     }
 }
 
 /// System Parachains.
 pub mod system_parachain {
-    /// Statemint parachain ID.
-    pub const STATEMINT_ID: u32 = 1000;
+    use xcm::latest::prelude::*;
+
+    /// Asset Hub parachain ID.
+    pub const ASSET_HUB_ID: u32 = 1000;
     /// Collectives parachain ID.
     pub const COLLECTIVES_ID: u32 = 1001;
+    /// Bridge Hub parachain ID.
+    pub const BRIDGE_HUB_ID: u32 = 1002;
+
+    frame_support::match_types! {
+        // System parachains from Paseo point of view.
+        pub type SystemParachains: impl Contains<MultiLocation> = {
+            MultiLocation {
+                parents: 0,
+                interior: X1(Parachain(
+                    ASSET_HUB_ID |
+                    COLLECTIVES_ID |
+                    BRIDGE_HUB_ID
+                )),
+            }
+        };
+    }
```

Comment:
Should we remove all of this?

Reply:
I don't think we need collectives there. But I would say there is no harm in leaving BridgheHub there.

Reply:
we can leave them indeed, it was just a cleaning proposal